### PR TITLE
Fix: Fixed regression pointed by #6650

### PR DESCRIPTION
**Pull request by leberus**
This is just a "dirty" workaround for this weekend, so r2 is not "broken". 
On Monday I'll check it out further :)

### CODE REVIEW

File: libr/debug/p/debug_native.c [hunk 1]
```diff
@@ -962,6 +962,7 @@ static RList *r_debug_native_map_get (RDebug *dbg) {
 	list->free = (RListFree)_map_free;
 	while (!feof (fd)) {
 		size_t line_len;
+		bool map_is_shared;
```
Comment (radare) [inline libr/debug/p/debug_native.c]: please initialize this variable

**Comment by SchumBlubBlub** [comment]
Something must be wrong, I still can't `db` on main and `iS` still show wrong address for rx segment.

**Comment by leberus** [comment]
I forgot to revert one line. I'll fix it now

El dia 03/02/2017 18:50, "SchumBlubBlub" <notifications@github.com> va
escriure:

**Comment by SchumBlubBlub** [comment]
Actually I took it back, I recompiled from libr/ instead of top so the bin wasn't updated. Is ok now afaict.

But hey, glad to help... :P

**Comment by leberus** [comment]
Yep, i just checked and actually i fixed everything (i thought i was
missing something) but not.
So should be fine now :D

**Comment by leberus** [comment]
Thanks for helping ;)

El dia 03/02/2017 19:24, "Oscar Salvador" <osalvador.vilardaga@gmail.com>
va escriure:

**Comment by SchumBlubBlub** [comment]
Thanks for fixing :+1:

**Pull request merged**